common/config/config.go

```diff
@@ -550,6 +550,8 @@ type (
 		Authorizer string `yaml:"authorizer"`
 		// Empty string for noopClaimMapper or "default" for defaultJWTClaimMapper
 		ClaimMapper string `yaml:"claimMapper"`
+		// Empty string unless using "opa" Authorizer. The HTTP(s) endpoint which temporal will invoke for authorization decisions.
+		OpaEndpoint string `yaml:"opaEndpoint"`
```

Comment:
I'd like to make a new `AuthorizerConfig any` field here so authorizers/claim mappers can define their own config, but I might refactor that later